scripts/generate_driver_wrappers.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+"""This script is required for the auto generation of the
+   psa_crypto_driver_wrappers.c file"""
+
+import sys
+import os
+import jinja2
+
+def render(template_path: str) -> str:
```

Comment:
I was wondering what is the purpose of setting the __annotations__ attribute at this point? Is that a requirement or a preference? 

As described in  the [PEP_483](https://www.python.org/dev/peps/pep-0483/) and [PEP_484 ](https://www.python.org/dev/peps/pep-0484/) specifications, my core concern is that it a very usefull but new feature, and would limit the possible systems where this script is deployed to anything running at least Python 3.5.

Reply:
Type annotations are a preference of mine. We don't require them, but I strongly encourage them.

Python <3.6 is no longer supported upstream. We officially require Python ≥3.6 (see `README.md`). We still need support for Python 3.5 until we migrate our testing away from Ubuntu 16.04. Type annotations work from  Python 3.4 onwards. They're already used in several of our scripts.